ci/accel/scikit-learn-tests/README.md

```diff
@@ -0,0 +1,59 @@
+# scikit-learn Acceleration Tests
```

Comment:
General comment on method here - you don't need a local copy of sklearn's git source to run the sklearn tests, they're packaged as part of sklearn itself. In any environment with `cuml` and `sklearn` both installed you can run the sklearn tests as:

```
py.test -p cuml.accel --pyargs sklearn  # or `sklearn.foo.tests.bar` for a specific test file
```

I have a slight preference for writing our test setup to use this style. This better isolates the installation of `sklearn` (and selection of version) from the test runner, and also avoids the need for extra state like `SCIKIT_LEARN_SOURCE_PATH` and friends.

Reply:
Ah nice, I wasn't aware of that. I'll update that.